h2o-automl/src/main/java/ai/h2o/automl/modeling/XGBoostSteps.java

```diff
@@ -241,7 +241,7 @@ protected ModelSelectionStrategy getSelectionStrategy() {
                     aml().eventLog().info(EventLogEntry.Stage.ModelSelection, "Applying learning rate search on best XGBoost: "+bestXGB._key);
                     XGBoostParameters xgBoostParameters = (XGBoostParameters) bestXGB._parms.clone();
                     XGBoostParameters defaults = new XGBoostParameters();
-                    xgBoostParameters._ntrees = 10000; // reset ntrees (we'll need more for this fine tuning)
+                    xgBoostParameters._ntrees = 10000; // reset ntrees (we'll need more for this fine-tuning)
```

Comment:
Just to get past the `Check Pull Request Metadata` test (it requires changes in java to be run).